### PR TITLE
GUI tutorials for citadel

### PR DESCRIPTION
**Pull request by maryaB-osr**
Signed-off-by: maryaB-osr <marya@openrobotics.org>

### CODE REVIEW

File: blueprint/Manipulating_models.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ With it, you can select an entity to display some of its attributes, including i
 The plugin also displays the gravitational and magnetic fields of your world.
 
 Expanding the `Pose` attribute will show the coordinate values for a selected object.
-The values will update as the object is moved around (translated or rotated).
+The values will update as the object is moved around (translated or rotated) while the simulation is playing.
```
Comment (JShep1) [inline blueprint/Manipulating_models.md:107]: Nit: playing -> running

File: citadel/Manipulating_models.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+You can customize snap values by clicking on the furthest-right magnet icon in the Transform Control plugin.
+
+![Custom snap values](img/custom_snap.png)
+
+Try holding `Shift` and `Ctrl` simultaneously to snap a model along the world frame is if isn't already aligned.
+
+## Component Inspector
+
+The Component Inspector plugin is included in `shapes.sdf`.
+With it, you can select an entity to display some of its attributes, including its pose, whether or not wind is acting on it, whether or not it's static, and more.
```
Comment (JShep1) [inline citadel/Manipulating_models.md:105]: Nit: it -> this plugin

File: citadel/Manipulating_models.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+The plugin also displays the gravitational and magnetic fields of your world.
+
+Expanding the `Pose` attribute will show the coordinate values for a selected object.
+The values will update as the object is moved around (translated or rotated) while the simulation is playing.
```
Comment (JShep1) [inline citadel/Manipulating_models.md:109]: Nit: playing -> running

File: citadel/Manipulating_models.md (hunk 2 above)
Comment (JShep1) [reply citadel/Manipulating_models.md:105]: Could you also make this change to the blueprint tutorial?

File: citadel/Manipulating_models.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+Like the View Angle plugin, the Align Tool is also not pre-loaded and will also have to be selected from the plugins button on the toolbar.
+
+The Align Tool aligns a model up with the center/origin of another model, or with something besides a model, like a link for example.
+It can't, however, align a link or anything other than the top-level model up with something else.
+
+With `Relative to First` selected, select two models using `Ctrl` + click to see the second model align to the first along either it's x, y or z axis.
+Hovering over the buttons for each axis will show a preview of where the model would end up if aligned along that axis.
+
+You can align more that two models, just select the others with `Ctrl` + click.
```
Comment (JShep1) [inline citadel/Manipulating_models.md:140]: that -> than, same for blueprint tutorial

File: citadel/hotkeys.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+
+
+  <table cellspacing='0'> <!-- cellspacing='0' is important, must stay -->
+
+  <tr><th>Action</th><th>Operation</th><th>Instruction</th></tr><!-- Table Header -->
+
+  <tr><th colspan="3" class="sub"><b>MODE</b></th></tr>
+
+  <tr><td>Enter Selection mode (default mode)</td><td>Esc</td><td>press Esc for mode to select models </td></tr>
```
Comment (JShep1) [inline citadel/hotkeys.md:13]: Maybe update this description to parallel the two below rows for rotation and translation? Same for blueprint

File: citadel/Manipulating_models.md (hunk 4 above)
Comment (maryaB-osr) [reply citadel/Manipulating_models.md:140]: There isn't an align tool for blueprint right?

Comment (JShep1) [reply citadel/Manipulating_models.md]: > There isn't an align tool for blueprint right?

Correct

**Pull request merged**